contrib/python/podman/podman/client.py

```diff
@@ -175,12 +175,12 @@ def __init__(self,
             if not System(self._client).ping():
                 raise ConnectionRefusedError(
                     errno.ECONNREFUSED,
-                    'Failed varlink connection "{}"'.format(address), address)
+                    ('Failed varlink connection "{}"').format(address))
         except FileNotFoundError:
             raise ConnectionError(
                 errno.ECONNREFUSED,
                 ('Failed varlink connection "{}".'
-                 ' Is podman service running?').format(address), address)
+                 ' Is podman socket or service running?').format(address))
```

Comment:
Additional information in Error message LGTM, again why remove data field in Error?